gips/data/prism/prism.py

```diff
@@ -35,14 +35,15 @@
 
 from gips.data.core import Repository, Asset, Data
 
-from gips.utils import settings, VerboseOut, List2File
+from gips.utils import settings, List2File
+from gips import utils
 
 from gippy import GeoImage
 
 from numpy import mean
 
 __author__ = "Ian Cooke <icooke@ags.io>"
-__version__ = '0.0.1'
+__version__ = '0.1.0'
```

Comment:
It isn't used in this driver at the moment, but it would be good if the metadata indicated both the gips and driver version...most of prism doesn't involve doing any actual processing.